framework/sandstone_ssl_rand.cpp

```diff
@@ -27,15 +27,15 @@ static void sandstone_rand_freectx(void *ctx)
 }
 
 /* Random number generator functions */
-static int sandstone_rand_instantiate(void *ctx, unsigned int strength, int prediction_resistance,
+static int sandstone_rand_instantiate(ossl_unused void *ctx, unsigned int strength, int prediction_resistance,
                                const unsigned char *pstr, size_t pstr_len,
                                const OSSL_PARAM params[])
 {
     ctx = reinterpret_cast<void *>(EVP_RAND_STATE_READY);
```

Comment:
The problem is not that they are unused; we don't have the compiler warning for unused parameters and variables enabled (we should, but don't).

The issue is a set-but-unused variable here. This statement is useless and gets removed from the execution.

Reply:
I removed the ctx set and also leaved the unused as well.

Reply:
You didn't push the update (did you forget the `-a` in `git commit --amend -a`?).

The `ossl_unused` is inconsistent because we're not using any of the other parameters. Either add to them all, or to none.

Reply:
Yeah, I totally forgot to amend my changes lol.

Regarding to the `ossl_unused`, I'm also fan of consistency so removing all appearances.